pkgs/os-specific/linux/libnss-mysql/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchFromGitHub, autoreconfHook, which, libmysqlclient }:
+
+stdenv.mkDerivation rec {
+  pname = "libnss-mysql";
+  version = "1.7.1";
+
+  src = fetchFromGitHub {
+    owner = "saknopper";
+    repo = "libnss-mysql";
+    rev = "v${version}";
+    sha256 = "1fhsswa3h2nkhjkyjxxqnj07rlx6bmfvd8j521snimx2jba8h0d6";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
+  buildInputs = [ libmysqlclient which ];
```

Comment:
Assuming that `which` is needed for https://github.com/saknopper/libnss-mysql/blob/1a30e1c8966e349569bb1957d5aacbe162e05231/m4/mysql.m4#L58 it should actually be in `nativeBuildInputs` or am I missing something?

Reply:
Fixed